Simulation.java

```diff
@@ -475,29 +474,6 @@ public void run() {
 			
 		}
 	}
-
-	public void writeDataCSV() throws FileNotFoundException {
```

Comment:
Any reason we're deleting this method? Was it not needed to perform your clustering?

Reply:
No, it's not needed to perform the clustering. We're using out.tips which it's already in csv format.

At first, I thought we were using out.timerseries to performing the clustering. It wasn't in csv format, so that's why I wrote this method.

Reply:
ohh got it! thanks!

Reply:
I just realized this method is actually needed to resolve this [issue](https://github.com/trvrb/antigen/issues/22)!